syntax = "proto3";


message ErrorResponse {
  string error_description = 1;
}